pkgs/development/libraries/libexif/default.nix

```diff
@@ -11,6 +11,20 @@ stdenv.mkDerivation rec {
     sha256 = "0mzndakdi816zcs13z7yzp7hj031p2dcyfq2p391r63d9z21jmy1";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "CVE-2020-0198.patch";
+      url = "https://github.com/libexif/libexif/commit/ce03ad7ef4e8aeefce79192bf5b6f69fae396f0c.patch";
+      sha256 = "0cy3kspggysmj7y15m2klsbq7zw3ir3bb1sw6bv4prnx2c5mb667";
```

Comment:
Both `sha256` are wrong for me?

Reply:
Must be caching. *sigh*

Will fix after dinner.